foundations/foundation-auth/src/main/java/org/apache/servicecomb/foundation/auth/rbac/TokenAuthHeaderProvider.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This class and others move to huawei-cloud/servicestage module, do not need create a new module.

Reply:
has removed to registry-service-center module, they has more closer relationship